sublime_syntaxes/Assembly x86.sublime-syntax

```diff
@@ -0,0 +1,67 @@
+%YAML 1.2
+---
+# http://www.sublimetext.com/docs/3/syntax.html
+name: Assembly x86 (NASM)
+file_extensions:
+  - asm
+  - inc
```

Comment:
Syntax updates is the reason why the submodules are there yeah.

I think doing an upstream PR and keeping submodules is the best way to make it work but some syntaxes are not updated for years so it's hard to say whether it's worth it. I'll think about it this weekend.